### ISSUE TITLE
Better handling of diff specs over previous version

### ISSUE THREAD

**Issue by tabatkins** [tabatkins/bikeshed#957]
The CSSWG occasionally maintains new versions of a spec as a "diff spec" on top of the previous version, when it's just being used to hold features that were punted from the previous level, and the previous level should still be considered the "mainline" development branch.

The sections that get punted often refer to sections that aren't present in the diff; they were left in the main spec as they're unchanged.  This accidentally breaks autolinking; Bikeshed very purposely prevents you from autolinking to other versions of the same spec, to catch errors, but here I explicitly *want* all those refs to go back to the previous version, without having to tag every single one of them with an explicit `spec` attribute.

I should add some way to indicate that a spec is a "diff" spec on top of another level, and more intelligently handle anchors in that case.

Not sure of the mechanics of this yet; it can get complicated, I think.

Also want some explicit indicators in the spec that it's a diff, so you don't accidentally forget to remove the diff indicator when you shift mainline dev to the new version.

**Comment by astearns** [comment]
It would be cool if bikeshed could generate a +1 level diff spec with all of the sections in place with boilerplate "refer to previous level" links. If we started diff specs with this content, we'd avoid problems noted here: https://github.com/w3c/csswg-drafts/issues/1653#issuecomment-371312217

**Comment by svgeesus** [comment]
I agree that this would help! More so when the earlier level is also a bikeshed spec, of course
